vendor/github.com/ethereum/go-ethereum/whisper/whisperv6/message.go

```diff
@@ -46,6 +46,8 @@ type MessageParams struct {
 	PoW      float64
 	Payload  []byte
 	Padding  []byte
+
+	self bool
```

Comment:
A comment would be nice. Also, why not stay consistent and use `Self bool`?

Reply:
If we use a public field it'll be marshalled, sent and then read by other peers as `Self:true`.

Reply:
You could use field tag `"-"` here., then marshalling is omitted.

Reply:
Hm, where is it marshaled? I thought it's just used as a collection of options to easily pass around.